src/main/java/com/google/devtools/build/lib/actions/FileContentsProxy.java

```diff
@@ -20,8 +20,8 @@
 
 /**
  * In case we can't get a fast digest from the filesystem, we store this metadata as a proxy to the
- * file contents. Currently it is two timestamps and a "node id". On Linux we use both ctime and
- * mtime and inode number. We might want to add the device number in the future.
+ * file contents. Currently it is two timestamps and a "node id". On Linux, macOS and Windows we use
+ * both ctime and mtime and inode number. We might want to add the device number in the future.
```

Comment:
It's actually ctime/mtime everywhere, but inode number only on Linux/MacOS (see getNodeId() in WindowsFileSystem.java).

Reply:
Updated, PTAL